TypeError: Cannot read property 'isFile' of undefined

```
mortar --output build/combined.mbtiles build/lines.mbtiles build/labels.mbtiles
/usr/local/lib/node_modules/tile-mortar/cli.js:64
      if (stats.isFile()) {
                ^

TypeError: Cannot read property 'isFile' of undefined
    at /usr/local/lib/node_modules/tile-mortar/cli.js:64:17
    at FSReqWrap.oncomplete (fs.js:152:21)
make: *** [build/combined.mbtiles] Error 1
```

--- a/cli.js
+++ b/cli.js
@@ -61,7 +61,9 @@ async.eachSeries(
   inputs,
   function(f, callback) {
     fs.stat(f, function(err, stats){
-      if (stats.isFile()) {
+      if(err) {
+        throw(err);
+      } else if (stats.isFile()) {
         inputMBTilesPath.push(f);
         callback();
       } else if (stats.isDirectory()) {

